libs/pages/elewa/home/src/lib/impact-section/impact-section.component.scss

```diff
@@ -0,0 +1,58 @@
+img{
+    height:60%;
+    width:40%;
+    border-radius: 2%;
+    object-fit: cover ;
+}
+
+*{
+    font-family: var(--elewa-group-website-dmsans-medium);
+}
+
+.buttons-div{
+   display: flex;
+   gap: 4em;  
+
+}
+
+.main{
+    display: flex;
+    padding: 6em;
+    gap: 100px;
+}
+
+
+
+@media screen and (max-width: 840px) {
+    * {
+        margin: 0;
+    }
+
+    .main{
+        display: inline-flex;
+        flex-direction: column;
```

Comment:
buttons are same width on the mobile design.

Reply:
![buttons](https://user-images.githubusercontent.com/85109484/219282731-07c501a2-bcf1-4efb-9e47-924e794e556d.PNG)


Changed Button Width For Smaller Screens